.github/BUILD.md

```diff
@@ -39,6 +39,8 @@ To build Umbraco, fire up PowerShell and move to Umbraco's repository root (the
 
     build/build.ps1
     
+If you only see a build.bat-file, you're probably on the wrong repository. If you switch to the correct repository (dev-v8) the file will appear and you can build it.
```

Comment:
If you change "wrong repository" to "wrong branch" then I'll be happy with this change!

Reply:
Fixed it... I guess :)